lib/write-error.js

```diff
@@ -1,25 +1,90 @@
 'use strict';
-var chalk = require('chalk');
+let chalk = require('chalk');
+
+function extractBroccoliError(error) {
```

Comment:
I wonder if we could improve this by creating a shared interface. Right now if the error object changes in `broccoli-middleware`, we're going to have to update this code.

Reply:
I believe something like that would be nice to have, but it can be done separately.